x-pack/test/observability_ai_assistant_api_integration/tests/knowledge_base/knowledge_base_setup.spec.ts

```diff
@@ -7,30 +7,58 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
-import { deleteKnowledgeBaseModel, createKnowledgeBaseModel } from './helpers';
+import {
+  deleteKnowledgeBaseModel,
+  createKnowledgeBaseModel,
+  TINY_ELSER,
+  deleteInferenceEndpoint,
+} from './helpers';
 
 export default function ApiTest({ getService }: FtrProviderContext) {
   const ml = getService('ml');
+  const es = getService('es');
   const observabilityAIAssistantAPIClient = getService('observabilityAIAssistantAPIClient');
 
   describe('/internal/observability_ai_assistant/kb/setup', () => {
-    it('returns empty object when successful', async () => {
+    it('returns model info when successful', async () => {
       await createKnowledgeBaseModel(ml);
       const res = await observabilityAIAssistantAPIClient
-        .editor({
+        .admin({
           endpoint: 'POST /internal/observability_ai_assistant/kb/setup',
+          params: {
+            query: {
+              model_id: TINY_ELSER.id,
+            },
+          },
         })
         .expect(200);
-      expect(res.body).to.eql({});
+
+      expect(res.body.service_settings.model_id).to.be('pt_tiny_elser');
+      expect(res.body.inference_id).to.be('ai_assistant_kb_inference');
+
       await deleteKnowledgeBaseModel(ml);
+      await deleteInferenceEndpoint({ es });
     });
 
-    it('returns bad request if model cannot be installed', async () => {
-      await observabilityAIAssistantAPIClient
-        .editor({
+    it('returns error message if model is not deployed', async () => {
+      const res = await observabilityAIAssistantAPIClient
+        .admin({
           endpoint: 'POST /internal/observability_ai_assistant/kb/setup',
+          params: {
+            query: {
+              model_id: TINY_ELSER.id,
+            },
+          },
         })
-        .expect(400);
+        .expect(500);
+
+      // @ts-expect-error
+      expect(res.body.message).to.include.string(
```

Comment:
Yes, because according to the types the body doesn't have a message. The reason the type is off, is because the endpoint throws and exception which is caught and handled by `supertest`.